cmd/buildctl/build.go

```diff
@@ -49,9 +53,36 @@ var buildCommand = cli.Command{
 			Name:  "frontend-opt",
 			Usage: "Define custom options for frontend",
 		},
+		cli.BoolFlag{
+			Name:  "no-cache",
```

Comment:
should we remove this, for now, to avoid confusion if it doesn't work yet?

Reply:
I put "not implemented" to the command line help string.
This will be soonly implemented.